trace/agent.js

```diff
@@ -42,6 +43,8 @@ function Agent(options) {
     if (options.reporter) {
         this.reporter = options.reporter;
     }
+
+    this.openTracer = options.openTracer || openTracing;
```

Comment:
is this needed?  I thought openTracer is defined on the channel. This agent could stay the old way